buildSrc/src/main/java/Dependencies.kt

```diff
@@ -30,7 +30,7 @@ object Versions {
     const val disklrucache = "2.0.2"
     const val leakcanary = "2.4"
 
-    const val mozilla_appservices = "63.0.0"
```

Comment:
Can we do this bump in a separate PR, to make sure nothing breaks? Or do a round of testing here, that works as well.

Reply:
Let's do that here since it's a breaking change from AS.

Reply:
Sure thing. Let's just make sure we do a full smoke test of A-S-depended functionality here - logins, fxa, sync, send tab, etc.